keymap/vim.js

```diff
@@ -411,11 +411,11 @@
       cfg = cfg || {};
       var scope = cfg.scope;
       if (!option) {
-        throw Error('Unknown option: ' + name);
+        return new Error('Unknown option: ' + name);
```

Comment:
Doesn't really seem necessary. We're just trying to signal that an option was missing. We could return a hash signalling error, but this changes the least of the original code without requiring try/catch blocks.